deploy-lighthouse-reth.py

```
@@ -0,0 +1,892 @@
+    print(f'Unsupported platform: {platform_name}')
+    exit(1)
+
+binary_arch=get_machine_architecture()
+platform_arch=get_computer_platform()
+
+# Change to the home folder
+os.chdir(os.path.expanduser("~"))
+
+if not args.network and not args.skip_prompts:
+    # Ask the user for Ethereum network
+    index = SelectionMenu.get_selection(valid_networks,title='Node Install Quickstart :: CoinCashew.com',subtitle='Installs Reth EL / Lighthouse BN / Lighthouse VC / MEVboost\nSelect Ethereum network:')
```

Ruff (0.8.2)

`SelectionMenu` may be undefined, or defined from star imports

(F405)